pwndbg/disasm/arch.py

```diff
@@ -244,7 +244,7 @@ def dump(self, instruction):
 
         for i, op in enumerate(ins.operands):
             rv.append('   operands[%i] = %s' % (i, ops.get(op.type, op.type)))
-            rv.append('       access   = %s' % (access.get(op.access, op.access)))
+            #rv.append('       access   = %s' % (access.get(op.access, op.access)))
```

Comment:
I got `AttributeError` and I couldn't find where it is created, so I removed it to make dump be usable.
What's this attribute do? It seems that it's not capstone attribute.